techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022 Normation SAS
+
+bundle agent rudder_system_update_campaign {
```

Comment:
you can't have several campaign at once ? like one for security every week and one for non security every month ?

Reply:
we should, i just initially copy-pasted the system-update technique